src/standard.yml

```diff
@@ -195,8 +195,49 @@ jobs:
           executor: << parameters.executor >>
 
       - run: mkdir -p ~/teaspoon
+
+      # Install Phantomjs if it is not present on this image
+      - restore_cache:
+          key: v1-phantomjs-2.1.1
+      - run:
+          name: Install phantomjs
+          command: |
+            if ! [ $(which phantomjs) ]; then
+              sudo curl --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-2.1.1
+            fi
+
+            sudo chmod ugo+x /usr/local/bin/phantomjs
+      - save_cache:
+          key: v1-phantomjs-2.1.1
+          paths:
+            - /usr/local/bin/phantomjs
+
       - run:
           command: |
             bundle exec teaspoon \
             --suppress-log \
             --format junit > ~/teaspoon/teaspoon.xml
+  check_annotate:
+    description: run rake annotate and check for changes
+    parameters:
+      executor:
+        type: executor
+      mysql_db_type:
```

Comment:
What is this? Why is "type" a boolean?

Reply:
I figured it out. I feel like we should be using `db_type` that is a string that's either `posgres` or `mysql`. That's probably for another time, though.